src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+@use "../utils/mixin" as *;
+
+.page {
+  @include center;
```

Comment:
You can wrap the calendar or make it an element of the page and use margin: 0 auto to center just the calendar